### PR TITLE
Monitoring: add Universal Profiling components to supportedBeatsCompo…

### PR DESCRIPTION
**Pull request by florianl**
…nents

## What does this PR do?

To be able to properly configure monitoring the Universal Profiling components require the information from Elastic Agent on how to configure everything.

### CODE REVIEW

File: internal/pkg/agent/application/monitoring/v1_monitor.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ const (
 var (
 	errNoOuputPresent          = errors.New("outputs not part of the config")
 	supportedMetricsComponents = []string{"filebeat", "metricbeat", "apm-server", "auditbeat", "cloudbeat", "fleet-server", "heartbeat", "osquerybeat", "packetbeat", "shipper"}
-	supportedBeatsComponents   = []string{"filebeat", "metricbeat", "apm-server", "fleet-server", "auditbeat", "cloudbeat", "heartbeat", "osquerybeat", "packetbeat"}
+	supportedBeatsComponents   = []string{"filebeat", "metricbeat", "apm-server", "fleet-server", "auditbeat", "cloudbeat", "heartbeat", "osquerybeat", "packetbeat", "pf-host-agent", "pf-elastic-collecor", "pf-elastic-symbolizer"}
```
Comment (cmacknz) [inline internal/pkg/agent/application/monitoring/v1_monitor.go:61]:
```suggestion
	supportedBeatsComponents   = []string{"filebeat", "metricbeat", "apm-server", "fleet-server", "auditbeat", "cloudbeat", "heartbeat", "osquerybeat", "packetbeat", "pf-host-agent", "pf-elastic-collector", "pf-elastic-symbolizer"}
```

I assume there is supposed to be a t in there. I don't think that we have a reasonable way to test this automatically besides just manually starting the agent.

Comment (inge4pres) [reply internal/pkg/agent/application/monitoring/v1_monitor.go:61]: good spot!

File: internal/pkg/agent/application/monitoring/process.go [hunk 2]
```diff
@@ -22,12 +22,13 @@ import (
...
-	apmPrefix         = "apm-server"
-	apmTypePrefix     = "apm"
-	fleetServerPrefix = "fleet-server"
+	componentIDKey         = "componentID"
+	metricsPathKey         = "metricsPath"
+	timeout                = 10 * time.Second
+	apmPrefix              = "apm-server"
+	apmTypePrefix          = "apm"
+	fleetServerPrefix      = "fleet-server"
+	profilingServicePrefix = "pf-elastic-"
```
Comment (inge4pres) [inline internal/pkg/agent/application/monitoring/process.go:31]: I think this one needs to be literally 
```suggestion
	profilingServicePrefix = "pf-elastic-symbolizer"
```

Comment (inge4pres) [reply internal/pkg/agent/application/monitoring/process.go:31]: nvm, resolved 😄 apologies...

**Review by cmacknz — APPROVED** [review]

**Pull request merged**